ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.7.yaml

```diff
@@ -86,22 +92,24 @@ releases:
       architecture: amd64
       product: ocp
       stream: nightly
-      version: "4.16"
+      version: "4.17"
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: gcp-ipi-f28
-  cron: 31 20 10 * *
+- as: gcp-ipi-f30
```

Comment:
We are using `f28` instead of `f30` to keep a consistent style.

Reply:
I was thinking about this, and set f30 for all EOL versions (older than 4.12), for those non-EOL version, I set frequency f28. Do we need to set different frequency for EOL and non-EOL versions? thoughts?

Reply:
f30 and f28 actually make no difference, and does not stand for EOL version, lxia explained in https://github.com/openshift/release/pull/53830#issuecomment-2199251651, to keep consistent, we are using f28 for now.

Reply:
done